boot/boot_test.go

```diff
@@ -315,6 +312,34 @@ func setupUC20Bootenv(c *C, bl bootloader.Bootloader, opts *bootenv20Setup) (res
 	}
 }
 
+func setupUC20MockBootloaderEnv(c *C, bl bootloader.Bootloader, opts *bootenv20Setup, cleanups *[]func()) {
```

Comment:
nitpick, this could also just return a restore function and then the caller appends that to `cleanups`, feels slightly more in line with the rest of the codebase here

Reply:
Good point, I've changed this now.